packages/react-native-session-replay/android/src/test/kotlin/com/datadog/reactnative/sessionreplay/DdSessionReplayImplementationTest.kt

```diff
@@ -30,18 +37,58 @@ internal class DdSessionReplayImplementationTest {
     @Mock
     lateinit var mockPromise: Promise
 
+    @Mock
+    lateinit var mockSessionReplay: SessionReplayWrapper
+
     @BeforeEach
     fun `set up`() {
-        testedSessionReplay = DdSessionReplayImplementation()
+        testedSessionReplay = DdSessionReplayImplementation { mockSessionReplay }
     }
 
     @AfterEach
     fun `tear down`() {
     }
 
     @Test
-    fun `M do nothing W enable()`() {
+    fun `M enable session replay W enable()`(
+        forge: Forge,
+        @Forgery privacy: SessionReplayPrivacy
+    ) {
+        // Given
+        val replaySampleRate = forge.aDouble(0.0, 100.0)
+        val sessionReplayConfigCaptor = argumentCaptor<SessionReplayConfiguration>()
```

Comment:
very minor: it is possible to use annotation to generate `double`.

```suggestion
    fun `M enable session replay W enable()`(
        @DoubleForgery(min = 0.0, max = 100.0) replaySampleRate: Double,
        @Forgery privacy: SessionReplayPrivacy
    ) {
        // Given
        val sessionReplayConfigCaptor = argumentCaptor<SessionReplayConfiguration>()
```